### DISCUSSION TITLE
What is the national sport of 🇹🇭?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#938] [Galaxy Trivia] [answered]
**Bonus:** What is the traditional band worn by athletes?

**Comment by peterbaccaro** [comment, marked as answer]
The national sport of Thailand is **Muay Thai. Also known as "The Art of Eight Limbs"**. 

Muay Thai is a combat sport that originated in Thailand and has a long history and cultural significance in the country. It is a martial art that involves the use of fists, elbows, knees, and shins in striking techniques. Muay Thai is widely practiced and enjoyed as both a competitive sport and a form of self-defense in Thailand, and it has gained international popularity as well.

The traditional band worn by athletes in Thailand is called a "**Mongkhon**". The Mongkhon is a headband typically made of fabric, silk, or satin, and it is considered a sacred item in Thai culture. It is believed to provide protection, good luck, and blessings to the wearer. In Muay Thai, the Mongkhon is worn during important matches or training sessions and is typically removed before the actual fight begins. It serves as a symbol of respect and tradition in Thai martial arts. The design of the Mongkhon can vary, but it often features intricate patterns and may have symbols or images representing Thai culture or spirituality.
